libstratum/EthStratumClient.cpp

```diff
@@ -559,38 +558,5 @@ bool EthStratumClient::submit(Solution solution) {
 			cnote << "Nonce:" << "0x" + nonceHex;
 		}
 		return true;
-	}
```

Comment:
Always called with a valid share. None of this was ever used.

Reply:
Good. Stratum part should assume the solutions are valid.